glossary.asciidoc

```diff
@@ -317,13 +313,12 @@ onion routing::
     The encrypted data is transmitted through a series of network nodes called onion routers, each of which peels away a single layer, uncovering the data's next destination.
     When the final layer is decrypted, the message arrives at its destination.
     The sender remains anonymous because each intermediary knows only the location of the immediately preceding and following nodes.
-    With the SPHINX Mix Format, the final destination also remains anonymous as only the previous router could see it but does not know if they are routing it to the final node or just the next hop.
-    https://en.wikipedia.org/w/index.php?title=Onion_routing&oldid=870849217
+       https://en.wikipedia.org/w/index.php?title=Onion_routing&oldid=870849217
```

Comment:
Why did you remove that sentence?

Reply:
Here is my comment from before (I'm thinking like a moderate techie rather than a hard-core techie):

My reasoning: A glossary should define and explain a term and not introduce new complexity. What is "an onion" and a "hop of the onion"? What is SPHINX Mix Format? I think it is good to have an example, but it should be understandable without having to find other terms in the Glossary.

However, as I said in the thread above, I can't gauge the relative importance of information I don't know enough about.  If it is key for the understanding of the glossary by a techie, then keep it in.